File: chariBE/src/main/java/com/macia/chariBE/model/PushNotification.java
package com.macia.chariBE.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class PushNotification {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer NOF_ID;

    @Column(length = 100)
    private String title;

    @Column(length = 1000)
    private String message;

    @Column(length = 30)
    private String topic;
}

File: chari/lib/models/push_notification.dart
class PushNotification {
  String topic;
  String title;
  String description;
  String message;
  int nof_ID;

  PushNotification(this.topic,this.title,this.description,this.message,this.nof_ID) {}

  PushNotification.fromJson(Map json)
      : topic = json['topic'],
        title = json['title'],
        description = json['description'],
        message = json['message'],
        nof_ID = json['nof_ID'];

}